src/content/developers/docs/nodes-and-clients/light-clients/index.md

```diff
@@ -0,0 +1,59 @@
+
+Ethereum rollups would also benefit from light clients. One of the big problems for rollups has been hacks targeting the bridges that allow funds to transfer from Ethereum layer-1 to a rollup. One vulnerability is the oracles that rollups use to detect that a user has made a deposit into the bridge. If an oracle feeds bad data, they could trick the rollup into thinking there was a deposit to the bridge and to incorrectly release funds. A light client embedded in the rollup could be used to protect against corrupted oracles because the deposit into the bridge could come with a proof that can be verified by the rollup before releasing any tokens. The same concept could also be applied to other interchain bridges.
+
+Light clients could also be used to upgrade Ethereum wallets. Instead of trusting data provided from an RPC provider, your wallet could directly verify the data being presented to you using an embedded light client. This would add security to your wallet. If your RPC provider was dishonest and provided you with incorrect data, the embedded light client could tell you!
+
+## What is the current state of light client development? {#current-state-of-development}
+
+There are several light clients in development, including execution, consensus and combined execution/consensus light clients. These are the light client implementations we know of at the time of writing this page:
+
+- [Lodestar](https://github.com/ChainSafe/lodestar/tree/unstable/packages/light-client): consensus light client in Typescript
+- [Helios](https://github.com/a16z/helios): combined execution and consensus light client in Rust
+- [Geth](https://github.com/ethereum/go-ethereum/tree/master/light): light mode for execution client (in development) in Go
+- [Nimbus](https://nimbus.guide/el-light-client.html): consensus light client in Nim
+
+To our knowledge none of these are considered production-ready yet.
+
+There is also a lot of work being done to improve the ways that light clients can access Ethereum data. Currently, light clients rely on RPC requests to full nodes using a client/server model, but in the future the data could be requested in a more decentralized way using a dedicated network such as the [Portal Network](https://www.ethportal.net/) that could serve the data to light clients using a peer-to-peer gossip protocol.
+
+Other [roadmap](/roadmap) items such as [Verkle trees](/roadmap/verkle-trees) and [statelessness](/roadmap/statelessness) will eventually bring the security guarantees of light clients equal to those of full clients.
```

Comment:
([trailing slashes for internal links](https://ethereum.org/en/contributing/style-guide/#internal-links))
```suggestion
Other [roadmap](/roadmap/) items such as [Verkle trees](/roadmap/verkle-trees/) and [statelessness](/roadmap/statelessness/) will eventually bring the security guarantees of light clients equal to those of full clients.
```